django/santropolFeast/member/models.py

```diff
@@ -36,11 +36,11 @@ class Meta:
     gender = models.CharField(
         max_length=1,
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
+            ('F', 'Female'),
+            ('U', 'Unknown'),
         ),
-        default=1
+        default=2
```

Comment:
I'm not sure on this default value:

 - When i create a member in the python shell, the member.gender value is u'2'
 - When i create a member in the admin panel, the default value in the dropdown is 'Male'